src/utils.js

```diff
@@ -91,31 +91,71 @@ export const getSpacingClasses = memoize(
   (values, type) => type + values.toString()
 )
 
-function getCSS(prop, value, base) {
+function getCSS(
+  prop: string,
+  value: number | string,
+  base: number
+): { [string]: number } {
+  const num = typeof value === 'number' ? value : parseFloat(value || 0)
```

Comment:
For consistency, also allow `marginTop` and similar properties to be used as a string. So if you can do `margin="1"` you should also be able to do `marginTop="1"` although it doesn't seem that common of a scenario